vms/job/views.py

```diff
@@ -53,9 +53,11 @@ def form_valid(self, form):
         end_date_event = event.end_date
         start_date_job = form.cleaned_data.get('start_date')
         end_date_job = form.cleaned_data.get('end_date')
+        job_name = form.cleaned_data.get('name')
+        flag = Job.objects.filter(name = job_name).exists()
```

Comment:
Here you are checking only on the basis of job name which is wrong because two jobs may have the same name. Rather u should focus on removing those jobs which have all their attributes same. Correct me if I am wrong @tapasweni-pathak @tonythomas01 @prachi1210 @MeepyMay .

Reply:
@mohitkyadav Read #73 . It also stated that all the details should not be same, rather than name.

Reply:
@anjali-dhanuka This PR fixes #330, and if all job names are unique then there's no problem in other details, two different jobs can be done simultaneously. Where's the problem, please correct me if I'm wrong.